app/__init__.py

```diff
@@ -30,7 +30,7 @@ def create_app(db):
     # Attach namespaces to api
     namespaces = config_obj.APP_NAMESPACES
     from .utils import init_namespace
-    from .namespaces import healthcheck_ns, data_provider_ns, cases_count_scraper_ns,\
+    from .namespaces import healthcheck_ns, data_provider_ns, cases_count_scraper_ns, \
```

Comment:
you don't actually need the additional space. when you create a long line and then click enter pycharm will add the back slash for you and it doesnt add the space